[D:1763013] Error: Can't send a remote File Path to a different remote channel from FoD Jenkins Plugin 7.2/8

pvivekdev opened the pull request:
[D:1763013] Error: Can't send a remote File Path to a different remote channel from FoD Jenkins Plugin 7.2/8

Cause:
A FilePath instance is tied to the remoting channel (agent or controller) where it was created. Using a FilePath on a different channel, Jenkins throws an IllegalStateException. Fix:
Copying the file ensures locality:
By copying the payload file to the agent's workspace, you guarantee that the file is available on the agent's filesystem, not just on the controller or another agent. Reconstructing FilePath ensures correct channel binding:

Commit:
[D:1763013] Error: Can't send a remote FilePath to a different remote channel from FoD Jenkins Plugin 7.2/8
Cause:
A FilePath instance is tied to the remoting channel (agent or controller) where it was created. Using a FilePath on a different channel, Jenkins throws an IllegalStateException.
Fix:
Copying the file ensures locality:
By copying the payload file to the agent's workspace, you guarantee that the file is available on the agent's filesystem, not just on the controller or another agent.
Reconstructing FilePath ensures correct channel binding:

Commit:
[D:1763013] Error: Can't send a remote FilePath to a different remote channel from FoD Jenkins Plugin 7.2/8
Cause:
A FilePath instance is tied to the remoting channel (agent or controller) where it was created. Using a FilePath on a different channel, Jenkins throws an IllegalStateException.
Fix:
Copying the file ensures locality:
By copying the payload file to the agent's workspace, you guarantee that the file is available on the agent's filesystem, not just on the controller or another agent.
Reconstructing FilePath ensures correct channel binding:

Commit:
[D:1763013] Error: Can't send a remote FilePath to a different remote channel from FoD Jenkins Plugin 7.2/8
Cause:
A FilePath instance is tied to the remoting channel (agent or controller) where it was created. Using a FilePath on a different channel, Jenkins throws an IllegalStateException.
Fix:
Copying the file ensures locality:
By copying the payload file to the agent's workspace, you guarantee that the file is available on the agent's filesystem, not just on the controller or another agent.
Reconstructing FilePath ensures correct channel binding:

Commit:
Fix: Release Id not loading the release related settings.

c0d3m0nky commented on src/main/webapp/ScanSettings.js:
```diff
@@ -132,15 +132,8 @@ class ScanSettings {
     }
 
     async loadEntitlementSettings(releaseChangedPayload) {
-        if (!this.uiLoaded) {
```
Why was this removed?

pvivekdev replied:
The (! this.uiLoaded)  is always set to false and the functional call to this.loadEntitlements becomes recursive which prevents the rest of the UI controls from loading, 

Have tested the Freestyle flow with the removal, Able to render the ui controls successfully.

Commit:
[D:1763013] Error: Can't send a remote File Path to a different remote channel from FoD Jenkins Plugin 7.2/8
Fix:
File Path made Transient
Removed CopyTo and construct the filePath using the Absolution Path
